demo/auth/index.js

```diff
@@ -0,0 +1,5 @@
+angular.module('wfm.auth', []);
```

Comment:
This should be moved out of the demo and renamed to be passport specific. 
We may try to have module to remove some common code and point customers to module instead of parts of the demo app which may be polluted with other non authentication specific code.